sql/migrations/20170614155841_uniqueDelegates.sql

```diff
@@ -0,0 +1,17 @@
+/*
+ * Recreate 'delegates' table to get rid of duplicated entries
+ */
+
+BEGIN;
+
+DELETE FROM delegates;
+
+INSERT INTO "delegates" ("username", "transactionId")
+SELECT m."username", t."id"
+FROM trs t
+LEFT JOIN mem_accounts m ON t."senderPublicKey" = m."publicKey"
+WHERE t.type = 2;
+
+ALTER TABLE "delegates" ADD CONSTRAINT "delegates_unique" UNIQUE ("username", "transactionId");
```

Comment:
We should enforce two checks - one for `username` and one for `transactionId`, because enter data like:
`('4miners.net', 1234), ('4miners.net', 12345)` should be not possible.